.github/workflows/rust.yml

```diff
@@ -69,6 +69,14 @@ jobs:
     - name: Run cargo fmt
       run: cargo fmt --check
 
+  cargo-doc:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: run cargo doc
+      run: RUSTDOCFLAGS="-D warnings" cargo doc --features derive,schema
```

Comment:
added this check to ci, as there're sometimes warnings, which signal, that the generated doc
won't look as expected, when rendered, and sometimes errors, which will prevent doc from being 
generated at release time altogether.

Reply:
This is a valuable piece of context worth leaving a comment in the file itself, so we don't lose it